gulpfile.js

```diff
@@ -68,46 +60,62 @@ function lint() {
     .pipe(gjslint.reporter('console'))
 }
 
-function bundle(browserify, env) {
-  return browserify
-    .transform(babelify)
-    .transform(envify, env)
-    .bundle()
-    .on('error', gutil.log.bind(gutil, 'Browserify Error'))
-    .pipe(source(jsFileName))
-    .pipe(buffer())
-    .pipe(sourcemaps.init({loadMaps: true}))
-      .pipe(uglify({
-        preserveComments: 'some'
-      }))
-      .on('error', gutil.log)
-    .pipe(sourcemaps.write('./'))
-    .pipe(gulp.dest('dist'));
+function bundle(format) {
+  return esperanto.bundle({
+    base: '.',
+    entry: entryFileName,
+  }).then(function(bundle) {
+    return bundle[format]({
+      strict: true,
+      sourceMap: 'inline',
```

Comment:
> If this works, it's pretty damn clever compared to @thejameskyle's version. Ping @jmeas.

I disagree. This increases the file size by including the entire source map in all built files rather than on demand (dev tools requesting it). @jmeas version is better